### PR TITLE
End2end - material page

### PR DESCRIPTION
**Pull request by Adamik10**
#### Link to issue
https://reload.atlassian.net/browse/DDFSOEG-309

#### Description
This PR adds material page to the user journey cypress end-to-end test. The test checks whether key data that a user would be focusing on during their user journey is rendered and whether clicking on the reservation button opens a modal.

#### Screenshot of the result
n/a

#### Checklist

- [x] My complies with [our coding guidelines](../docs/code_guidelines.md).
- [x] My code passes our static analysis suite. If not then I have added a comment explaining why this change should be exempt from the code standards and process.
- [x] My code passes our continuous integration process. If not then I have added a comment explaining why this change should be exempt from the code standards and process.

#### Additional comments or questions
Contains #216 + #217

### CODE REVIEW

File: cypress/e2e/user-journey.cy.ts [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+const patronData = require("../../wiremock/src/mappings/work/data/fbs/patron.json");
+
+describe("User journey", () => {
+  it("Shows search suggestions & redirects to search result page", () => {
+    cy.createMapping({
+      request: {
+        method: "POST",
+        url: "/opac/graphql",
+      },
+      response: {
```
Comment (spaceo) [inline cypress/e2e/user-journey.cy.ts:16]: There is one thing I would like to talk with you about looking at these mappings. I find it unnecessary to create the same mappings as we are using in the Pa11y and Lighthouse tests.

File: cypress/e2e/user-journey.cy.ts [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+  it("Shows material page & can open reservation modal", () => {
+    materialPageMocking();
+
+    cy.visit("/work/work-of:870970-basis:25245784")
+      .contains("Harry Potter og Fønixordenen")
+      .get('[data-cy="material-header-buttons-physical"]')
+      .should("contain", "Reserve bog")
+      .click()
+      .get('[data-cy="modal"]')
+      .should("contain", "Harry Potter og Fønixordenen");
```
Comment (spaceo) [inline cypress/e2e/user-journey.cy.ts:141]: I think we should use getBySel here

**Review by spaceo — CHANGES_REQUESTED** [review]
We need to talk about the mappings. There is one comment about the actual test I would like you to look at :)

**Comment by Adamik10** [comment]
Became a part of #216

**Pull request closed without merging**